Test/TestGui/TestGui.gd

```diff
@@ -7,18 +7,24 @@ const file_name = "res://Test/TestParser/Timeline.rk"
 @onready var menu_container = $MenuContainer
 
 func _ready():
-	Rakugo.connect("say",Callable(self,"_on_say"))
-	Rakugo.connect("step",Callable(self,"_on_step"))
-	Rakugo.connect("ask",Callable(self,"_on_ask"))
-	Rakugo.connect("menu",Callable(self,"_on_menu"))
-	Rakugo.connect("notify",Callable(self,"_on_notify"))
+	Rakugo.add_custom_regex("HW", "^hello_world$")
+	Rakugo.connect("sg_say",Callable(self,"_on_say"))
```

Comment:
If you use Godot V4 signal you can write
Rakugo.sg_say.connect(_on_say)

https://docs.godotengine.org/en/stable/getting_started/step_by_step/signals.html#complete-script

Reply:
Okey I will update those calls.

Reply:
Just fixed with my last commit @theludovyc